installer/roles/local_docker/templates/docker-compose.yml.j2

```diff
@@ -34,6 +34,7 @@ services:
     {% endif %}
     {% if ssl_certificate is defined %}
       - "{{ ssl_certificate +':/etc/nginx/awxweb.pem:ro' }}"
+      - "{{ docker_compose_dir }}/nginx.conf:/etc/nginx/nginx.conf:ro"
```

Comment:
Shouldn't this be outside the `if ssl_certificate` block?

Reply:
Done ;)